<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>471</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Podatki o datotekah za zdru≈æitev</string>
   </property>
   <widget class="QLabel" name="NewBorn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Datum [DDMMYYYY]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Date">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pasma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alelni format</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ZIP datoteka</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Breed">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="AlleleFormat">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ZipFile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Prebrskaj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZipFileShow">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ParentalVerification">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>242</green>
         <blue>180</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>236</red>
         <green>248</green>
         <blue>217</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>109</red>
         <green>121</green>
         <blue>90</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>145</red>
         <green>161</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>242</green>
         <blue>180</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>236</red>
         <green>248</green>
         <blue>217</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>242</green>
         <blue>180</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>236</red>
         <green>248</green>
         <blue>217</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>109</red>
         <green>121</green>
         <blue>90</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>145</red>
         <green>161</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>242</green>
         <blue>180</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>236</red>
         <green>248</green>
         <blue>217</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>109</red>
         <green>121</green>
         <blue>90</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>242</green>
         <blue>180</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>236</red>
         <green>248</green>
         <blue>217</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>109</red>
         <green>121</green>
         <blue>90</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>145</red>
         <green>161</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>109</red>
         <green>121</green>
         <blue>90</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>109</red>
         <green>121</green>
         <blue>90</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>242</green>
         <blue>180</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>242</green>
         <blue>180</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>242</green>
         <blue>180</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title">
    <string>Preverjanje star≈°evstva</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="ParentalVerificationYN">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>99</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>≈†tevilo SNPov</string>
    </property>
   </widget>
   <widget class="QComboBox" name="NoSNPs">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Kaj ≈æeli≈° storiti?</string>
   </property>
   <widget class="QCheckBox" name="mergeAsk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Zdru≈æi s prej≈°njimi datotekami</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="removeZipAsk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Izbri≈°i originalen ZIP file</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>601</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Direktoriji z datotekami in programi</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Zaƒçasni direktorij</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Direktorij ZIP datotek</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Zdru≈æeni PLINK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pedda Row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tempDir">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Prebrskaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZipDir">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Prebrskaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MergedDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Prebrskaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PeddarowDir">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Prebrskaj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PeddarowDirShow">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MergeDirShow">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZipDirShow">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tempDirShow">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZanardiDirShow">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ZanardiDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Prebrskaj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Zanardi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CodeDir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CodeDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Prebrskaj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CodeDirShow">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="DoEverything">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>520</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Izvedi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
